jwt_proxy/api.py

```diff
@@ -67,13 +96,16 @@ def validate_jwt(relative_path):
         )
     except jwt.exceptions.ExpiredSignatureError:
         return jsonify(message="token expired"), 401
-
-    response_content = proxy_request(
-        req=request,
-        upstream_url=f"{current_app.config['UPSTREAM_SERVER']}/{relative_path}",
-        user_info=decoded_token.get("email") or decoded_token.get("preferred_username"),
-    )
-    return response_content
+    
+    # TODO: call new function here to dynamically load a filter call dependent on config; hardwired for now
+    if scope_filter(request, decoded_token):
```

Comment:
probably best to exit early (with the `return jsonify...` as you have).  i fear we'll get a deep nest of it clauses when additional filters come online.

Reply:
thanks Paul! was going to mention something similar; sometimes called a "guard clause"

Reply:
Nice catch, thanks both!